examples/with_winit/src/main.rs

```diff
@@ -245,6 +262,7 @@ fn main() -> Result<()> {
     env_logger::init();
     let args = Args::parse();
     let scenes = args.args.select_scene_set(|| Args::command())?;
+    let base_color = args.args.get_base_color()?;
```

Comment:
I wasn't too happy with this. The only reason I put it here was to propagate the error but it's probably not egregious to just unwrap the result inside `run`.

Reply:
> I just want to check ,clap doesn't handle Option<Color> automatically, does it? I think they use FromStr, so it might just work.

Actually this does work pretty well. I used `value_parser` instead of FromStr though I imagine that would work too. The output looks even nicer now. PTAL.